pkg/scheduler/framework/types.go

```diff
@@ -39,6 +39,61 @@ import (
 
 var generation int64
 
+// ActionType is an integer to represent one type of resource change.
+// Different ActionTypes can be bit-wised to compose new semantics.
+type ActionType int64
+
+// Constants for ActionTypes.
+const (
+	Add    ActionType = 1 << iota // 1
+	Delete                        // 10
+	// UpdateNodeXYZ is only applicable for Node events.
+	UpdateNodeUnschedulable // 100
+	UpdateNodeAllocatable   // 1000
+	UpdateNodeLabel         // 10000
+	UpdateNodeTaint         // 100000
+	UpdateNodeCondition     // 1000000
+
+	All ActionType = 1<<iota - 1 // 1111111
+
+	// Use the general Update type if you don't either know or care the specific sub-Update type to use.
+	Update = UpdateNodeUnschedulable | UpdateNodeAllocatable |
```

Comment:
`Update ActionType`?

Reply:
The type is inferrable, golint gave a "Type can be omitted" warning.

Reply:
oh, from the right-hand-side, makes sense.